paper.tex

```diff
@@ -63,18 +63,15 @@ \subsection{Generative models}
 
 Diffusion models achieve state-of-the-art synthesis results on image data by decomposing the image formation process into sequential applications of denoising autoencoders. Due to its work directly in pixel space, they are very computational expensive. To lower the computational costs, the Stable Diffusion model (v1.5) is applied in a latent space of powerful pretrained autoencoders. This works without losing quality and flexibility. \cite{Rombach_2022_CVPR}
 
-As a second image generative model we use \textbf{OpenDalle} from dataautogpt3 (\url{https://huggingface.co/dataautogpt3/OpenDalle}). OpenDalle, as it is based on the Dall-E model, works by modelling text and image tokens as a single data stream. This is achieved by training a transformer to autoregressively model those tokens into the resulting stream.
-
+As the second image generative model we use \textbf{OpenDalle} from dataautogpt3 (\url{https://huggingface.co/dataautogpt3/OpenDalle}). OpenDalle, as it is based on the Dall-E model, works by modelling text and image tokens as a single data stream. This is achieved by training a transformer to autoregressively model those tokens into the resulting stream.
 This is done in two steps: Firstly, The images get compressed in a $32 \times 32$ grid of image tokens to reduce the context size of the transformer. Secondly, BPE-encoded text tokens are concatenated with the image tokens to model a joint distribution over text and image tokens with the training of an autoregressive transformer.
```

Comment:
If we remove the newline here, I would also remove the line-break afterwards, so we merge these two paragraphs

Reply:
dalle and midjourney are not the same model why would you merge them

Reply:
No I mean merging line 66 and line 67 to one paragraph.

Reply:
they are one paragraph i think no?

Reply:
Yeah, but they still contain a line-break. This does not change the resulting report, so we can also just leave it as is.